Develop

MelnichukMykola opened the pull request:
https://futdevelop.github.io/react_todo-app-with-api/

MelnichukMykola committed:
made solution

MelnichukMykola committed:
made solution

MelnichukMykola committed:
made solution

MelnichukMykola committed:
made solution

etojeDenys reviewed (changes requested):
you need to implement loading logic for deleting, adding, clear completed, toggle all like in the working example

MelnichukMykola committed:
made solution

VitaliyBondarenko1982 reviewed (changes requested):
About problem with loader replied in chat.
Try to fix tests and linter warnings.

MelnichukMykola committed:
made solution

MelnichukMykola committed:
made solution

MelnichukMykola committed:
made solution

MelnichukMykola committed:
made solution

volodymyr-soltys97 reviewed (changes requested):
Hi, I have some question for you.
Tell me, did you really write the assignment yourself? It looks like the code was copied from another student's assignment that I checked yesterday, because the functions, variables have the same name, even the same comments, although the initial version of your code was completely different.
https://github.com/mate-academy/react_todo-app-with-api/pull/1638/files

MelnichukMykola committed:
made solution

MelnichukMykola commented:
> Hi, I have some question for you. Tell me, did you really write the assignment yourself? It looks like the code was copied from another student's assignment that I checked yesterday, because the functions, variables have the same name, even the same comments, although the initial version of your code was completely different. https://github.com/mate-academy/react_todo-app-with-api/pull/1638/files


Yeah, I really borrow some ideas, but I don't thought to copy everything. Now I have changed all code as I could. I hope on your acception
- Added animation
- Changed some enums logic
- Reduced code but still effective

vadiimvooo reviewed (approved):
Well done
